src/manifest.c

```diff
@@ -589,8 +589,18 @@ struct manifest *load_mom(int version)
 	string_or_die(&filename, "%s/%i/Manifest.MoM", state_dir, version);
 	string_or_die(&url, "%s/%i/Manifest.MoM", content_url, version);
 	if (!download_and_verify_signature(url, filename)) {
-		printf("WARNING!!! FAILED TO VERIFY SIGNATURE OF Manifest.MoM\n");
+		if (!force_ignore_unverified_signature) {
+			printf("FAILED TO VERIFY SIGNATURE OF Manifest.MoM. Can proceed if"
+				"  --force-continue-with-unverified-signature used, but system"
+				" security WILL BE COMPROMISED\n");
+			return NULL;
+		} else {
+			printf("FAILED TO VERIFY SIGNATURE OF Manifest.MoM. Operation proceeding due to"
+				"  --force-continue-with-unverified-signature, but system security WILL BE"
+				" COMPROMISED\n");
+		}
```

Comment:
The WILL BE COMPROMISED statement strikes me as overly confident and assuming.

"FAILED TO VERIFY SIGNATURE OF Manifest.MoM." followed by the exit in the first condition strikes me as sufficient.

In the second case I can see reason for a "...proceeding with forced" message.

Generally I've come to prefer more terse printf's as we've experienced a lot of confusion with verbose output on the console.  It is a standard paradigm to see a failure and then call the command with --help to see if there's any sort of force override flag for folks who are inclined to go there.